src/payload.rs

```diff
@@ -24,47 +24,41 @@ use {
     anyhow::Result,
     base64::prelude::{Engine, BASE64_STANDARD},
     serde::{Deserialize, Serialize},
-    solana_client::rpc_config::RpcSendTransactionConfig,
-    solana_sdk::transaction::VersionedTransaction,
+    solana_sdk::{
+        clock::Slot, commitment_config::CommitmentLevel, pubkey::Pubkey,
+        transaction::VersionedTransaction,
+    },
     solana_transaction_status::UiTransactionEncoding,
     std::str::FromStr,
     thiserror::Error,
 };
 
 #[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct RpcSendTransactionConfigWithBlockList {
-    #[serde(flatten)]
-    pub config: Option<RpcSendTransactionConfig>,
-    /// base58-encoded pubkeys
-    pub blocklist_pdas: Option<Vec<String>>,
+pub struct RpcSendTransactionConfig {
+    #[serde(default)]
+    pub skip_preflight: bool,
+    #[serde(default)]
+    pub skip_sanitize: bool,
+    pub preflight_commitment: Option<CommitmentLevel>,
+    pub encoding: Option<UiTransactionEncoding>,
+    pub max_retries: Option<usize>,
+    pub min_context_slot: Option<Slot>,
+    #[serde(default, deserialize_with = "deserialize_forwarding_policies")]
+    pub forwarding_policies: Vec<Pubkey>,
 }
```

Comment:
This needs to be optional, so that's why I recommended to just have two interfaces.